src/android/java/io/jxcore/node/StreamCopyingThread.java

```diff
@@ -9,18 +8,18 @@
 /**
  * Created by juksilve on 5.6.2015.
  */
-public class StreamCopyingThread extends Thread {
+class StreamCopyingThread extends Thread {
 
 
     interface CopyThreadCallback
     {
-        public void StreamCopyError(StreamCopyingThread who, String error);
+        void StreamCopyError(StreamCopyingThread who, String error);
     }
 
-    CopyThreadCallback callback = null;
+    private CopyThreadCallback callback = null;
 
-    String TAG = "StreamCopyingThread";
-    boolean mStopped = false;
+    private String TAG = "StreamCopyingThread";
+    private boolean mStopped = false;
 
     private static final int DEFAULT_BUFFER_SIZE = 1024 * 4;
 
```

Comment:
with current re-structuring, the IDE gives error if I try setting them final, thus not attempting to change.